app/scripts/modules/core/pipeline/config/triggers/webhook/webhookTrigger.module.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.trigger.webhook', [
+    require('../../../../../core/config/settings.js'),
+  ])
+  .config(function (pipelineConfigProvider) {
+    pipelineConfigProvider.registerTrigger({
+      label: 'Webhook',
+      description: 'Executes the pipeline on a Webhook POST',
+      key: 'webhook',
+      controller: 'WebhookTriggerCtrl as ctrl',
+      controllerAs: 'vm',
+      templateUrl: require('./webhookTrigger.html'),
+      popoverLabelUrl: require('./webhookPopoverLabel.html'),
+    });
+  })
+  .controller('WebhookTriggerCtrl', function (trigger, $scope) {
```

Comment:
Thanks @anotherchrisberry   I'll aim to get a new PR in over the weekend.
I did get as far a looking through gate source code to find where the echo mappings are defined but haven't found anything yet. Should I be able to test firing a webhook into gate somehow?